common.yaml

```diff
@@ -27,6 +27,7 @@ services:
         environment:
             POSTGRES_USER: "$DB_USER"
             POSTGRES_PASSWORD: "$DB_PASSWORD"
+            POSTGRES_INITDB_ARGS: "--data-checksums --locale=C --encoding=UTF8"
```

Comment:
setting `--locale=C` will affect users with other locales?

Reply:
Yes, it will make their databases perform properly.  The problem you usually have with alpine postgres is it uses en_US.utf8, which for Odoo breaks btree for text search.

In any case, from v12, I see Odoo is forcing C collation.  I could change it to just LC_COLLATE if that is what's wanted.

Reply:
I just read the docs and its hard to see the difference between setting locale and LC_COLLATE.  https://www.postgresql.org/docs/9.6/static/locale.html

if locale is set it sets LC_COLLATE.  If LC_COLLATE is set, it sets locale, which will then set LC_CTYPE and other LC vars unless specified.

Any specific setting for the other LC settings can be overidden in INITDB_ARGS.

Reply:
Probably the only way of doing in container would also solve the unaccent issue.

If database does not exist, CREATE DATABASE WITH LC_COLLATE etc.  Then you can add extensions.  Before all the auto install stuff of modules.  Picking LC_COLLATE is a one-shot thing for a new db.

idk to be honest, I just ported those 3 initdb args from my standard postgres compose file, but left out the non essential ones.

Reply:
I wouldn't like to create the database here manually.

Odoo, from v9+, automatically creates it. IMHO if we have to fix the DB creation process somewhere, it's there, directly upstream.